.zenodo.json

```diff
@@ -15,6 +15,10 @@
       "orcid": "0000-0002-6533-164X"
     },
     {
+      "name": "Madeleine Snyder",
+      "affiliation": "Harvard Medical School, Harvard University, Cambridge, Massachusetts"
+    },
+    {
```

Comment:
Could you move this to just above Russ Poldrack? Outside contributors are listed in order of contribution, starting at Feilong Ma.

Also, we haven't generally included city and state in the affiliation. For consistency, I'd suggest dropping it. And do you have an [ORCID](https://orcid.org/)? It doesn't take much time to get one, and they help disambiguate researchers with similar names, as well as persist across changes in institution/name.

Reply:
Sorry for the ordering issue, it's my bad. @madeleinesnyder please fix the ordering and add your ORCID if available/possible.